### PR TITLE
chore: finishing touches before release

### PR DESCRIPTION
**Pull request by thepassle**

### CODE REVIEW

File: packages/eslint-plugin-lit-a11y/README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 Accessibility linting plugin for lit-html.
 
-Most of the rules are ported from [eslint-plugin-jsx-a11y](https://github.com/jsx-eslint/eslint-plugin-jsx-a11y).
+Most of the rules are ported from [eslint-plugin-jsx-a11y](https://github.com/jsx-eslint/eslint-plugin-jsx-a11y), and made to work with [lit-html](https://lit-html.polymer-project.org/) templates.
```
Comment (bennypowers) [inline packages/eslint-plugin-lit-a11y/README.md:5]:
```suggestion
Most of the rules are ported from [eslint-plugin-jsx-a11y](https://github.com/jsx-eslint/eslint-plugin-jsx-a11y), and made to work with HTML and [lit-html](https://lit-html.polymer-project.org/) templates.
```

**Review by bennypowers — APPROVED** [review]

File: packages/eslint-plugin-lit-a11y/README.md (hunk 1 above)
Comment (thepassle) [reply packages/eslint-plugin-lit-a11y/README.md:5]: it doesnt work with plain HTML templates though :p

Comment (bennypowers) [reply packages/eslint-plugin-lit-a11y/README.md:5]: it does in lit-html templates

the intention here was more to the "custom elements" part of HTML though, e.g. props -> attributes, HOCs -> slots

Comment (thepassle) [reply packages/eslint-plugin-lit-a11y/README.md:5]: maybe 

> and made to work with [lit-html](https://lit-html.polymer-project.org/) templates and custom elements.

then ?

I dont want to give people the idea that this eslint plugin works for plain html

Comment (bennypowers) [reply packages/eslint-plugin-lit-a11y/README.md:5]: > made to work with custom elements and ...
?

Comment (thepassle) [reply packages/eslint-plugin-lit-a11y/README.md:5]: I'd put lit-html first because its the primary usecase

Comment (bennypowers) [reply packages/eslint-plugin-lit-a11y/README.md:5]: updated the suggestion

**Pull request merged**